src/Workspaces/Core/Portable/ExternalAccess/UnitTesting/Api/UnitTestingSolutionExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
+{
+    internal static class UnitTestingSolutionExtensions
+    {
+        public static int UnitTesting_GetWorkspaceVersion(this Solution solution)
+            => solution.WorkspaceVersion;
+
+        public static UnitTestingSolutionStateWrapper GetUnitTestingSolutionStateWrapper(this Solution solution)
```

Comment:
You control this API name, but if you want to simplify it you can:

```suggestion
        public static UnitTestingSolutionStateWrapper GetState(this Solution solution)
```